omod-2.0/src/test/java/org/openmrs/module/webservices/rest/web/v1_0/controller/openmrs2_0/ConceptAttributeTypeController2_0Test.java

```diff
@@ -85,6 +85,16 @@ public void shouldListAllConceptAttributeTypes() throws Exception {
 		Assert.assertEquals(getAllCount(), Util.getResultsSize(result));
 	}
 
+	@Test
+   	public void shouldPurgeConceptAttributeType() throws Exception {
+    	final String uuid = "9516cc50-6f9f-11e0-8414-001e378eb67f";
```

Comment:
It would be great to follow our naming convention for constants.

Reply:
Alright let me do that @kdaud

Reply:
@kdaud  I have added to your review and made another commit.